tests/robot_tests.cpp

```diff
@@ -46,3 +46,15 @@ TEST_CASE("pick_ik::get_link_indexes") {
         CHECK(tip_link_indexes->at(0) == 0);
     }
 }
+
+TEST_CASE("pick_ik::Robot::from") {
+    auto const robot_model = make_rr_model();
+    auto* const jmg = robot_model->getJointModelGroup("group");
+    auto const tip_link_indexes =
+        pick_ik::get_link_indexes(robot_model, {"b"})
+            .or_else([](auto const& error) { throw std::invalid_argument(error); })
+            .value();
+    auto const robot = pick_ik::Robot::from(robot_model, jmg, tip_link_indexes);
```

Comment:
What about testing the values of all the other public fields in this struct?

Reply:
![image](https://user-images.githubusercontent.com/2490389/192557475-28675298-25f9-4ae4-ad44-6c6fe2888136.png)

Here are the default values in the variables structs. I didn't write a test of these because I can't imagine this is the behavior we want.

Reply:
As in, you don't want to these initial values because you don't want to keep the behavior this way?

Reply:
I feel like this part of the API (the robot struct) is still part of the cruft from bio_ik. I don't want to write tests for code I want to eliminate.